Tools/Scripts/webkitpy/benchmark_runner/data/patches/webserver/MotionMark.patch

```diff
@@ -47,30 +114,37 @@ index d45155e..b10635a 100644
 +
 +    didFinishLastIteration: function()
 +    {
-+        // submit result to server
-+        var results = JSON.stringify(this._computeResultsReports());
-+        var xhr = new XMLHttpRequest();
-+        xhr.open("POST", "/report");
-+        xhr.setRequestHeader("Content-type", "application/json");
-+        xhr.setRequestHeader("Content-length", results.length);
-+        xhr.setRequestHeader("Connection", "close");
-+        xhr.onreadystatechange = function() {
-+            if (xhr.readyState == XMLHttpRequest.DONE && xhr.status == 200) {
-+                closeRequest = new XMLHttpRequest();
-+                closeRequest.open("GET", "/shutdown");
-+                closeRequest.send()
-+            }
-+        }
-+        xhr.send(results);
++       console.log("[DEBUG] Finished last iteration")
++       // submit result to server
++       var results = JSON.stringify(this._computeResultsReports());
++       console.log(results)
```

Comment:
Ditto.

Reply:
Updated in revised commit, forgot to take this out beforehand wups